.github/workflows/build.yml

```
@@ -270,6 +280,44 @@ jobs:
           rm -rf temp
           mv signed temp
 
+      - name: Generate VSA
+        if: env.shouldSign == 'true'
+        env:
+          AWS_KMS_REGION:  ${{ job.build.steps.auth.with.args.aws-region }}
+          AWS_KMS_KEY_ARN: ${{ job.build.steps.sign.env.AWS_KMS_KEY_ARN }}
+        run: |
+          args+=(
+            --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
+
+            # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
+            "$BASHBREW_IMAGE_SIGNER"
+
+            # verify is called here to generate a VSA (https://slsa.dev/spec/v1.0/verification_summary)
+            verify
+
+            -D # debug output
+
+            --platform="$BASHBREW_ARCH"
+            --vsa=true # whether or not to generate a VSA from verification output
+
+            --tuf-source=oci  # how to pull TUF data for policy and key material (oci or http)
+            --tuf-meta-source="$BASHBREW_TUF_META"  # TUF metadata URI
+            --tuf-targets-source="$BASHBREW_TUF_TARGETS"  # TUF targets URI
+
+            --policy-id="$BASHBREW_SIGNING_POLICY"  # policy stored in TUF for sbom/provenance verification
+
+            # kms key used to sign VSA
+            --kms="AWS"
+            --kms-region="$AWS_KMS_REGION"
+            --kms-key-ref="$AWS_KMS_KEY_ARN"
+
+            --image=oci:///doi-build/signed  # subject image to verify attestations and generate VSA for
```

```suggestion
            --image=oci:///doi-build/image # subject image to verify attestations and generate VSA for
```
same as above, the image can be whatever the build step outputs